core/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -649,7 +643,8 @@ public void onMarkerPick(final MarkerPickResult markerPickResult, final float po
         mapView.post(new Runnable() {
           @Override public void run() {
             if (markerPickResult != null) {
-              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker()));
+              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker(),
```

Comment:
Comment on the documentation above for `setMarkerPickListener`.

Because of the distinction between a `Marker` (backed by `Tangram.MapData`) and a `BitmapMarker` (backed by `Tangram.Marker`), I think the documentation should clarify this to be a `BitmapMarker` and not a `Marker`.